tests/__init__.py

```diff
@@ -14,15 +16,20 @@
     reason="needs async support (added in Python 3.5)")
 
 
+def call_vulture(args, **kwargs):
+    return subprocess.call(
+        [sys.executable, '-m', 'vulture'] + args, cwd=REPO, **kwargs)
+
+
 def check(items_or_names, expected_names):
-    if isinstance(items_or_names, set):
-        # items_or_names is a set of strings.
-        assert items_or_names == set(expected_names)
-    else:
-        # items_or_names is a list of Item objects.
+    try:
+        # items_or_names is a collection of Item objects.
         names = sorted(item.name for item in items_or_names)
```

Comment:
Only put the line that can fail into the try-block, everything else should go into the else-block.